sdk/docker-solana/localnet.sh

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
```

Comment:
I personally feel like we shouldn't need to maintain this and we don't need to make it too easy to use docker for development. The docker file is probably sufficient for those who really want to use docker